freqtrade/persistence/models.py

```diff
@@ -179,6 +180,7 @@ def update_from_ccxt_object(self, order):
         self.amount = order.get('amount', self.amount)
         self.filled = order.get('filled', self.filled)
         self.average = order.get('average', self.average)
+        self.initial_average = order.get('average', self.initial_average)
```

Comment:
This is on the order object, which corresponds to one exchange order.

The average of the order will not change - so this seems like a pointless column.

Reply:
It is changed when profit is given to the previous orders.